robottelo/datafactory.py

```diff
@@ -379,21 +379,21 @@ def valid_http_credentials(url_encoded=False):
             u'http_valid': False,
         },
         {
-            u'login': gen_string('alpha', gen_integer(1, 100)),
+            u'login': gen_string('alpha', gen_integer(1, 512)),
             u'pass': gen_string('alpha'),
             u'quote': False,
             u'http_valid': False,
         },
         {
-            u'login': gen_string('alphanumeric', gen_integer(1, 100)),
+            u'login': gen_string('alphanumeric', gen_integer(1, 512)),
             u'pass': gen_string('alphanumeric'),
             u'quote': False,
             u'http_valid': False,
         },
         {
-            u'login': gen_string('utf8', gen_integer(1, 100)),
+            u'login': gen_string('utf8', gen_integer(1, 50)),
```

Comment:
@elyezer I tested with a varying number to reach an acceptable boundary, assuming that url_enconding will increase the length.

Look this example:

```python
In [23]: len("䑧㭿䑧㭿䑧㭿䑧㭿䑧㭿䑧㭿䑧㭿䑧㭿䑧㭿")
Out[23]: 54

In [24]: len(quote_plus("䑧㭿䑧㭿䑧㭿䑧㭿䑧㭿䑧㭿䑧㭿䑧㭿䑧㭿"))
Out[24]: 162

In [25]: print quote_plus("䑧㭿䑧㭿䑧㭿䑧㭿䑧㭿䑧㭿䑧㭿䑧㭿䑧㭿")
%E4%91%A7%E3%AD%BF%E4%91%A7%E3%AD%BF%E4%91%A7%E3%AD%BF%E4%91%A7%E3%AD%BF%E4%91%A7%E3%AD%BF%E4%91%A7%E3%AD%BF%E4%91%A7%E3%AD%BF%E4%91%A7%E3%AD%BF%E4%91%A7%E3%AD%BF 
```

Having 
150 for user + 150 for password + owner + url + reponame
50 was the safest size I found on tests.

BTW: there is this [RFE] opened https://bugzilla.redhat.com/show_bug.cgi?id=1325001 if able to pass credentials as headers we should pass any value up to 8k (web server HTTP header limit)